.gitignore

```diff
@@ -0,0 +1,4 @@
+*.txt*
+*books_dict* 
```

Comment:
Ah cool!  Yes, this is another way to make sure you don't include data.  Very useful also when we get to virtual environments.  I usually add the env/ directory to make sure it doesn't get pushed to GitHub.